### PR TITLE
First GSoC'21 blog post

### PR DESCRIPTION
**Pull request by Yash-10**
@astrojuanlu I added my first blog post. Please let me know if it looks alright.

Thanks!

### CODE REVIEW

File: content/articles/2021-06-06-poliastro-google-summer-of-code-2021.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Title: GSoC 2021, the journey begins!
+Date: 2021-06-06 11:00 IST
+Category: GSOC
+Tags: GSOC, GSOC21, poliastro
+slug: 2021-06-06-poliastro-google-summer-of-code-2021
```
Comment (astrojuanlu) [inline content/articles/2021-06-06-poliastro-google-summer-of-code-2021.md]: Please remove the date from the slug - we've been adding it for some time, but it's redundant :) See the last published article

File: content/articles/2021-06-06-poliastro-google-summer-of-code-2021.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+lang: en
+Author: Yash
+
+
+# The initial days…
+
+Hi there! I am Yash, and I will be spending this summer with Poliastro as part of GSoC 2021 to add some event detection capabilities. It was around December when I first learned about Poliastro, and it gradually got me fascinated by orbital mechanics.
```
Comment (astrojuanlu) [inline content/articles/2021-06-06-poliastro-google-summer-of-code-2021.md]: I know it's silly, but poliastro is not capitalized 😄

**Review by astrojuanlu — COMMENTED** [review]
Added two nitpicks, but other than that this looks perfect 😻

File: content/articles/2021-06-06-poliastro-google-summer-of-code-2021.md (hunk 1 above)
Comment (Yash-10) [reply content/articles/2021-06-06-poliastro-google-summer-of-code-2021.md]: Thanks!

File: content/articles/2021-06-06-poliastro-google-summer-of-code-2021.md (hunk 2 above)
Comment (Yash-10) [reply content/articles/2021-06-06-poliastro-google-summer-of-code-2021.md]: Oh! I forgot to lowercase it earlier..Thanks

**Review by astrojuanlu — APPROVED** [review]

**Comment by astrojuanlu** [comment]
Thanks a lot!

**Pull request merged**